### PR TITLE
feat: patient file overview

### PR DESCRIPTION
**Pull request by RyanWennekes**
image
image

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,7 +23,9 @@ services:
     expose:
       - "3000"
     environment:
-      - MESSAGE_QUEUE=amqp://messagebus:5672
+      - MESSAGE_BUS_URI=amqp://messagebus:5672
+      - MESSAGE_BUS_EXCHANGE_NAME=storeInteraction
+      - MESSAGE_BUS_QUEUE_NAME=storeInteractionQueue
```
Comment (RyanWennekes) [inline docker-compose.yml:28]: Dit was ik vergeten mee te pushen naar dev in story 189.

File: docker-compose.yml [hunk 2]
```diff
@@ -86,7 +88,7 @@ services:
     container_name: generaldatadb
     image: mcr.microsoft.com/mssql/server:2019-latest
     environment:
-      MSSQL_SA_PASSWORD: GeneralDataDBPass12345!@
+      MSSQL_SA_PASSWORD: DbPasswordDev12345!@
```
Comment (RyanWennekes) [inline docker-compose.yml:91]: Als je de containers opnieuw opspint: eerst de volumes verwijderen!

**Review by ntjhal — APPROVED** [review]
Code looks good to me! Pretty sure the PatientController will give merge conflicts with the other PR that's currently opened #43 but we'll see when the first one is merged tot dev

**Review by MerijnMonfils — APPROVED** [review]
👍🏻

**Pull request merged**